src/main/java/melky/worldflags/WorldFlagsMode.java

```diff
@@ -37,10 +37,10 @@
 @RequiredArgsConstructor
 public enum WorldFlagsMode
 {
-	CHAT_CHANNEL(ScriptID.FRIENDS_CHAT_CHANNEL_REBUILD, WidgetInfo.FRIENDS_CHAT_LIST, 1, "^W.*$", "W"),
-	CLAN_CHANNEL(4397, WidgetInfo.CLAN_MEMBER_LIST, 1, "^W.*$", "W"),
-	GUEST_CHANNEL(4399, WidgetInfo.CLAN_GUEST_MEMBER_LIST, 1, "^W.*$", "W"),
-	FRIENDS(ScriptID.FRIENDS_UPDATE, WidgetInfo.FRIEND_LIST_NAMES_CONTAINER, 2, "^World\\s?.*$", "World "),
+	CHAT_CHANNEL(ScriptID.FRIENDS_CHAT_CHANNEL_REBUILD, WidgetInfo.FRIENDS_CHAT_LIST, 1, "^W[0-9][0-9][0-9]\\b", "W"),
+	CLAN_CHANNEL(4397, WidgetInfo.CLAN_MEMBER_LIST, 1, "^W[0-9][0-9][0-9]\\b", "W"),
+	GUEST_CHANNEL(4399, WidgetInfo.CLAN_GUEST_MEMBER_LIST, 1, "^W[0-9][0-9][0-9]\\b", "W"),
+	FRIENDS(ScriptID.FRIENDS_UPDATE, WidgetInfo.FRIEND_LIST_NAMES_CONTAINER, 2, "^World\\s[0-9][0-9][0-9]\\b", "World "),
```

Comment:
```suggestion
	CHAT_CHANNEL(ScriptID.FRIENDS_CHAT_CHANNEL_REBUILD, WidgetInfo.FRIENDS_CHAT_LIST, 1, "^W[0-9]{1,3}\\b", "W"),
	CLAN_CHANNEL(4397, WidgetInfo.CLAN_MEMBER_LIST, 1, "^W[0-9]{1,3}\\b", "W"),
	GUEST_CHANNEL(4399, WidgetInfo.CLAN_GUEST_MEMBER_LIST, 1, "^W[0-9]{1,3}\\b", "W"),
	FRIENDS(ScriptID.FRIENDS_UPDATE, WidgetInfo.FRIEND_LIST_NAMES_CONTAINER, 2, "^World\\s[0-9]{1,3}\\b", "World "),
```
Best to check for 1-3 digits. I know that RS3 uses double digit worlds but I can't remember if we were able to get the region of the worlds but I am pretty sure there was a reason I left it at .*